server/client_ca_test.go

```
@@ -223,20 +224,27 @@ func TestClientCADerivations(t *testing.T) {
+
+		binaryCacheURL := fmt.Sprintf("s3://%s?endpoint=http://%s&region=eu-west-1&store=%s", testService.Bucket, endpoint, storeDir)
 
 		testEnv := append(os.Environ(),
 			"AWS_ACCESS_KEY_ID=minioadmin",
 			"AWS_SECRET_ACCESS_KEY="+testMinioServer.secret,
 		)
 
 		// Try to copy from our cache
-		cmd := exec.CommandContext(ctx, "nix", "--extra-experimental-features", "nix-command", "copy",
+		// Pass the full absolute path - NIX_STORE_DIR env var handles the rest
+		copyArgs := []string{"--extra-experimental-features", "nix-command flakes ca-derivations", "copy",
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gofumpt)